src/dokan/README.md

```diff
@@ -0,0 +1,42 @@
+About
```

Comment:
This documentation should be in `doc/cephfs/` as an `.rst` with appropriate formatting. I would expect a basic usage section similar to `ceph-fuse`/`kernel`. I would also expect a small blurb about the windows driver here: https://docs.ceph.com/en/latest/cephfs/mount-prerequisites/

Reply:
Following this, I'd encourage a blog post on ceph.io. Would you be interested?

Reply:
> This documentation should be in `doc/cephfs/` as an `.rst` with appropriate formatting. I would expect a basic usage section similar to `ceph-fuse`/`kernel`. I would also expect a small blurb about the windows driver here: https://docs.ceph.com/en/latest/cephfs/mount-prerequisites/

Makes sense. TBH, I was thinking about having some developer notes in ``src/dokan/README.md`` and the install and usage guide in ``README.windows.rst`` but it's quite redundant ATM. I'll start preparing a `doc/cephfs/` doc.

Reply:
> Following this, I'd encourage a blog post on ceph.io. Would you be interested?

That would be great, thanks! We actually got WP access yesterday so we'll prepare a blog post.